doc.go

```diff
@@ -1,4 +1,2 @@
-/*
-EMV Payment Code Encoder/Decoder for Go
-*/
-package main
+// Package EMV Payment Code Encoder/Decoder for Go.
```

Comment:
I prefer to comment as following 👀 	
```suggestion
// Package emvcode is the EMV Payment Code Encoder/Decoder for Go.
```

Reply:
@ymt2 Fixed.